core/src/main/java/nl/nn/adapterframework/pipes/JwtPipe.java

```diff
@@ -46,20 +48,23 @@
 import nl.nn.adapterframework.parameters.ParameterList;
 import nl.nn.adapterframework.parameters.ParameterValueList;
 import nl.nn.adapterframework.stream.Message;
+import nl.nn.adapterframework.util.AppConstants;
 import nl.nn.adapterframework.util.CredentialFactory;
 
 /**
  * Creates a JWT with a shared secret using the HmacSHA256 algorithm.
  *
  * @ff.parameter {@value #SHARED_SECRET_PARAMETER_NAME} overrides attribute <code>sharedSecret</code>
- * 
+ *
  * @author Niels Meijer
  * @since 7.9
  */
 @Category("Basic")
 public class JwtPipe extends FixedForwardPipe {
 	private static final String SHARED_SECRET_PARAMETER_NAME = "sharedSecret";
 
+	private @Setter(AccessLevel.PACKAGE) boolean jwtAllowWeakSecrets = AppConstants.getInstance().getBoolean("application.security.jwt.allow.weak.secrets", false);
```

Comment:
Ik denk dat het mooier is om 'category' en 'variablenaam' te scheiden.

```suggestion
	private @Setter(AccessLevel.PACKAGE) boolean jwtAllowWeakSecrets = AppConstants.getInstance().getBoolean("application.security.jwt.allowWeakSecrets", false);
```

Reply:
Goed punt! Ga ik aanpassen.